tests/test_config.py

```diff
@@ -6,6 +6,12 @@
 from dxr.exceptions import ConfigError
 
 
+def fail(message):
+    """Raise an AssertionError along with given message.
+    """
+    assert False, message
```

Comment:
Calls to `assert` are no good, because python -O optimizes them out. (I've actually been bitten by that at Votizon, where we used -O for some reason.) For this reason and for consistency with the rest of nose (which doesn't do that, IIRC), let's `raise AssertionError(message)` or `ok_(False, message)` instead.